redis/src/push_manager.rs

```diff
@@ -0,0 +1,95 @@
+use crate::{PushKind, RedisResult, Value};
+use arc_swap::ArcSwap;
+use std::collections::HashMap;
+use std::sync::mpsc::SyncSender;
+use std::sync::Arc;
+
+/// Holds information about received Push data
+#[derive(Debug, Clone)]
+pub struct PushInfo {
+    /// Push Kind
+    pub kind: PushKind,
+    /// Data from push message
+    pub data: Vec<Value>,
+
+    /// Connection address to distinguish connections
+    pub con_addr: Arc<String>,
```

Comment:
when is this needed?

Reply:
It's for client side cache, to use later on